vision/vae_mnist/README.md

```diff
@@ -1,6 +1,15 @@
-# Variational Autoencoder(VAE)
+# Variational Autoencoder (VAE)
+
+<img src="..\vae_mnist\docs\vae.png" width="500">
```

Comment:
Similar comments about image sourcing and reference formats for this file. Seems like the path is still incorrect here too?